PWGDQ/dielectron/core/AliDielectronVarManager.h

```diff
@@ -1257,6 +1258,7 @@ inline void AliDielectronVarManager::FillVarESDtrack(const AliESDtrack *particle
 
 
   values[AliDielectronVarManager::kTPCsignal]=particle->GetTPCsignal();
+  values[AliDielectronVarManager::kTPCsignalTunedOnData]=particle->GetTPCsignalTunedOnData();
```

Comment:
This should be fine. What about the AOD tracks? Is there a tune on Data tpc signal also for AOD tracks? Please implement it also there.